qoffer-infrastructure/src/main/groovy/life/qbic/portal/qoffer2/customers/CustomerDbConnector.groovy

```diff
@@ -436,4 +443,31 @@ class CustomerDbConnector implements CreateCustomerDataSource, UpdateCustomerDat
 
     return generatedKeys[0]
   }
+
+  /**
+   * Searches for a person and returns the matching ID from the person table
+   *
+   * @param person The database entry is searched for a person
+   * @return the ID of the database entry matching the person
+   */
+  @Override
+  int getPersonId(Person person) {
```

Comment:
Again, i think it is a mistake to make this method public, and i don't see the necessity for it.